vendors/carbonblack.json

```diff
@@ -1,8 +1,19 @@
 {
   "name": "Carbon Black",
+  "productCapabilities": ["anti-malware", "av", "edr", "endpoint detection", "endpoint response", "etdr"],
```

Comment:
I wonder if we could align the specific capabilities to the function. For example, anti-malware, anti-virus would be device-protect, but the others would align differently.

Reply:
Hrm. Maybe say more about that? I don't yet see the benefit of spelling out that alignment. One nice thing about the current structure is that you should be able to say `Find Vendor with productCapability ~= 'anti-malware'` and get a list of all vendor-related controls that provide that capability.

Reply:
I think one potential issue with that approach is that the list of things that are "device-protect" is nearly infinite, and frequently new categories are discovered.  Knowing the full list of what things are possible to be queried then becomes a matter of tribal knowledge, instead of using the CDM to identify those products.  "device-protect" will give us whatever fits into that category, regardless of the name instead of having to know specifically to list "anti-malware, anti-virus (or whether they're "antimalware" or "antivirus" instead), or whatever other product acronym gets developed in order for the producer to differentiate their product/service somehow.

Reply:
I think eventually we may publish our mapping of industry-standard capability terms to CDM asset-functions. Being able to point to this mapping as a fairly objective "standard" will help us justify our classification of each vendor's positions in the matrix, as well as serve as a "lingua franca" for those new to the matrix.